tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -1800,17 +1816,24 @@ def window(self, size, shift=None, stride=1, drop_remainder=False):
     {'a': [1, 2]}
     {'a': [3, 4]}
 
+    If this dataset returns elements in a deterministic order, then the
```

Comment:
I think we can remove this paragraph since the same could be said about just about every transformation: the output is deterministic if and only if the input is deterministic.

Reply:
Removed this paragraph in commit 84bc9b4